style.css

```diff
@@ -899,3 +950,689 @@ input {
+    display: none;
+  }
+
+  #pro-images-1 {
+    top: -429px;
+    margin: auto;
+    left: 328px;
+    background-image: url(./images/Img\ Placeholder.png);
+    background-size: 91%;
+    width: 410px;
+  }
+
+  .button2 {
+    margin-top: 254px;
+    margin-left: 5px;
+    width: 325px;
+    height: 52px;
+  }
+
+  .buttontwo {
+    display: none;
+  }
+
+  .button-text2 {
+    left: 3px;
+    position: relative;
+  }
+
+  .professional-art-text {
+    margin-top: 20px;
+    padding-left: 14px;
+    padding-top: 77px;
+  }
+
+  #pro-images-2 {
+    top: -916px;
+    left: 687px;
+    height: 345px;
+    background-size: 85%;
+    width: 371px;
+    background-image: url(./images/page2card2.png);
+  }
+
+  .pro-para-text {
+    width: 243px;
+    margin-bottom: 50px;
+    height: 79px;
+    font-size: 15px;
+  }
+
+  #pro-images-3 {
+    top: -1358px;
+    left: 1026px;
+    height: 402px;
+    width: 316px;
+    background-size: 100%;
+    background-image: url(./images/card3.png);
+  }
+
+  .card1 {
+    margin-left: -43px;
+  }
+
+  .card2 {
+    margin-left: 23px;
+  }
+
+  .card3 {
+    margin-left: 127px;
+  }
+
+  #pro-images-4 {
+    top: -1450px;
+    left: 345px;
+    height: 345px;
+    background-size: 94%;
+    width: 338px;
+    background-image: url(./images/probackground.png);
+  }
+
+  .pro-desktop {
+    display: flex;
+    font-family: "Crete Round", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff;
+    margin-top: 27px;
+  }
+
+  .professional-text {
+    display: none;
+  }
+
+  .professional-text-4 {
+    padding-top: 9px;
+  }
+
+  #pro-images-5 {
+    top: -1892px;
+    left: 689px;
+    height: 363px;
+    background-size: 82%;
+    width: 388px;
+    background-image: url(./images/page2card2.png);
+  }
+
+  .pro-div {
+    display: grid;
+    justify-content: center;
+    width: 100%;
+    margin-left: 337px;
+    height: 1px;
+  }
+
+  #pro-images-6 {
+    top: -2352px;
+    left: 1028px;
+    height: 423px;
+    width: 316px;
+    background-size: 100%;
+    background-image: url(./images/card3.png);
+  }
+
+  .about-me {
+    width: 100%;
+    height: 1195px;
+    top: 385%;
+    background-image: url(./images/about-me-background.png);
+    background-size: 46%;
+    background-position-x: right;
+    background-position-y: 85px;
+  }
+
+  .about-me-div {
+    width: 566px;
+    height: 426px;
+    margin-top: -26px;
+    margin-left: 73px;
+  }
+
+  .about-me-text {
+    padding-top: 80px;
+    font-size: 72px;
+    padding-left: 92px;
+  }
+
+  .about-me-para {
+    padding-left: 90px;
+    width: 420px;
+    font-size: 14px;
+    height: 148px;
+    line-height: 32px;
+  }
+
+  .button3 {
+    margin-left: 91px;
+    margin-top: 0;
+  }
+
+  .line {
+    width: 650px;
+    position: relative;
+    top: 135px;
+  }
+
+  .about-me-section {
+    display: flex;
+    justify-content: center;
+    position: relative;
+    top: 205px;
+  }
+
```

Comment:
- [ ] please make sure the about me section's language part is centered on the page currently :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230709065-17369c4a-7418-47cd-a1ae-f99a43a46e2c.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/230709118-a3f268d3-3712-4f9c-9188-6ce1341ffaf7.png)